tests/test_main.cpp

```diff
@@ -270,11 +273,13 @@ struct GodotTestCaseListener : public doctest::IReporter {
 				SceneTree::get_singleton()->get_root()->set_embedding_subwindows(true);
 			}
 
+#ifdef TOOLS_ENABLED
 			if (name.find("[Editor]") != -1) {
 				Engine::get_singleton()->set_editor_hint(true);
 				EditorPaths::create();
 				EditorSettings::create();
 			}
+#endif // TOOLS_ENABLED
```

Comment:
Unsure if we should error here when not using a tools build just to warn, didn't add anything right now but can do if desired